src/transactions/ParallelApplyUtils.cpp

```diff
@@ -896,7 +897,10 @@ OpParallelApplyLedgerState::upsertEntry(LedgerKey const& key,
                std::this_thread::get_id(),
                liveEntryExistedAlready ? "already-live" : "new",
                xdr::xdr_to_string(key, "key"));
-    mOpEntryMap[key] = entry;
+
+    // Weird syntax avoids redundant map lookup
+    auto& mapEntry = mOpEntryMap[key] = entry;
+    mapEntry->lastModifiedLedgerSeq = ledgerSeq;
```

Comment:
I think there's a scenario still broken with RO ttl bumps. If an RO bump has not been observed by the end of a thread, we [flush](https://github.com/stellar/stellar-core/blob/611395c2df77ddb9666db12dccfd26ca081fc566/src/transactions/ParallelApplyUtils.cpp#L680) it at the end of the thread. This would not get updated with the right `lastModifiedLedgerSeq`.

Reply:
Good catch, I'll add RO tests for meta. `ExtendFootprintOp` uses RO ttl bumps right?

Reply:
I've pushed your tests, along with a check and fix for RO TTL lastModifiedLedgerSeq.